app/assets/javascripts/enroll_camera.js

```diff
@@ -0,0 +1,42 @@
+$(document).ready(function(){
+  
+
+  $('#shutter-button').on('click', function(){
+    take_snapshot()
+    removeShutter()
+    addSubmit()
+  })
+
+  $('div.button').on('click', '#submit-button', function(e) {
+    e.preventDefault()
+    let image = $('#my_camera').children('img').attr('src')
+    $.ajax({
+      type: "POST",
+      url: '/enrollments',
+      data: {image: image},
+      success: function(data){
+
+      }
+    })
+  })
+
+  $('div.button').on('click', '#reset-button', function(e) {
+    e.preventDefault()
+    location.reload()
+  })
+
+  function take_snapshot() {
+    Webcam.snap( function(data_uri) {
+      document.getElementById('my_camera').innerHTML = '<img src="'+data_uri+'"/>';
+    } );
+  }
+
+  function removeShutter(){
+    $('#shutter-button').remove()
+  }
+
+  function addSubmit(){
+    $('div.container .button').append('<button class="btn btn-primary" id="submit-button">Submit Photo</button>')
```

Comment:
to make this even cleaner, i'd save the selector as a variable and extract out the two constant html pieces.. something like:

```js
function submitButton() {
  return '<button class="btn btn-primary" id="submit-button">Submit Photo</button>'
}

function addSubmit() {
  const $button = $('div.container .button')
  $button.append(submitButton())
}
```